zh-CN/mapping_dev.zh_CN

```diff
@@ -1,44 +1,44 @@
 :*"Cancel"=取消
-:*"Add Branch"=创建分支
+:*"Add Branch"=添加分支
 :*"Discard"=丢弃
-:*"Git-Flow"=Git工作流
+:*"Git-Flow"=Git 流
 :*"Commit"=提交
 :*"Branch"=分支
 :*"Check Out"=检出
 :*"Merge"=合并
 :*"Pull"=拉取
 :*"Push"=推送
 :*"Rebase"=变基
-:*"Save Stash"=贮藏
+:*"Save Stash"=保存储藏
 :*"Stage"=暂存
 :*"Sync"=同步
 :*"Unstage"=取消暂存
 :*"Deselect All"=取消全选
 :*"Select All"=全选
 :*"Output"=输出
-:*"Repositories"=版本库
-:*"Add Tag"=创建标签
-:*"Apply Stash"=应用贮藏
-:*"Working Tree"=工作副本
+:*"Repositories"=储存库
+:*"Add Tag"=添加标签
```

Comment:
Between "添加" and "创建",  I prefer to use "创建" because it is a kind of action that is out of nothing.  If it is literal translation, then it will be very boring.

I will not point out one by one in other places, please confirm and update.

Reply:
Creating（ "创建"） here is more explicit than adding（ "添加"） semantics